packages/arb-token-bridge-ui/src/components/App/AppProviders.tsx

```diff
@@ -13,6 +13,10 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { createConfig } from '@lifi/sdk'
 import { INTEGRATOR_ID } from '../../pages/api/crosschain-transfers/lifi'
 
+import { ArbitrumIndexerProvider } from '../../../../indexer-provider'
```

Comment:
This doesn't exist in the repo. I was testing it locally by manually adding the package into the project. Needs to be handled.